block1_bugrace_intro/README.md

```diff
@@ -2,7 +2,7 @@
 
 Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
 Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
-<a href="www.dn.se" target="_blank">test</a>
+<br><a href="http://www.dn.se" target="_blank">test</a>
```

Comment:
Anledningen till att uppgifterna ligger på Github är för att det ska bli enklare för er att tillsammans med andra ta fram uppgifter, det gör att ni kan släppa in volontärer på GitHub som föreslår förändringar eller nya uppgifter utan att de behöver få tillgång till Umbraco login (där ni vill hålla accessen begränsad av flera anledningar). 

Men nackdelen med Github är att det blir svårt att göra speciallösningar med annat utseende, till exempel boxar med inramning osv utan att hacka in HTML, men det blir lätt lite kladdigt och svåradministrerat att blanda markdown och HTML. 



Palle


> On 4 dec. 2015, at 09:28, tinavincent <notifications@github.com> wrote:
> 
> In block1_bugrace_intro/README.md:
> 
> > @@ -2,7 +2,7 @@
> >  
> >  Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
> >  Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
> > -<a href="www.dn.se" target="_blank">test</a>
> > +<br><a href="http://www.dn.se" target="_blank">test</a>
> 
> Heading 3 har väl ingen färgmarkering vad jag minns? Det Lisa vill ha är ett avsnitt som är extra synligt, tex i form av färgmarkering, bakgrundsfärg, ram etc. Har ni några andra fiffiga saker som passar bättre så använder vi den gärna! Berätta gärna mer om er strategi med markdown vs html så förstår jag den biten lite bättre också. Inte jobbat med markdown tidigare. Varför vill man undvika html i sidorna? Konverteras det automatiskt och isf hur ofta? Kan jag pusha på så att det går snabbare på ngt sätt när jag vill testa en ändring? Mvh /Tina Vincent
> …
> —
> Reply to this email directly or view it on GitHub.
>